ui/src/pages/projects/add/index.tsx

```diff
@@ -0,0 +1,195 @@
+import { unwrapResult } from '@reduxjs/toolkit';
+import classNames from 'classnames/bind';
+import { Formik, FormikHelpers, FormikProps } from 'formik';
+import Link from 'next/link';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import * as Yup from 'yup';
+
+import ImageUploadField from 'src/components/Form/ImageUploadField';
+import SelectField from 'src/components/Form/SelectField';
+import TextField from 'src/components/Form/TextField';
+import DashboardLayout from 'src/containers/DashboardLayout';
+import { RootState } from 'src/duck';
+import { fetchOrganization, OrganizationState } from 'src/duck/organization';
+import { createProject } from 'src/duck/project';
+
+import style from './style.module.scss';
+
+import { AuthState } from 'src/duck/auth';
+
+import { getUsersOrganizationId } from 'src/entities/User/selectors';
+
+const c = classNames.bind(style);
+
+const Header: React.FC = () => (
+  <div className={c('header')}>
+    <span>
+      <Link href="/projects">
+        <a>Projects > </a>
+      </Link>
+      Create
+    </span>
+  </div>
+);
+
+const AddProject: React.FC<FormikProps<FormValues>> = (p) => {
+  const dispatch = useDispatch();
+
+  let userOrganizations = [];
+  const { user } = useSelector<RootState, AuthState>((state) => state.auth);
+  const orgId = getUsersOrganizationId();
+
+  const { organizations, isFetching: isFetchingOrganizations } = useSelector<
+    RootState,
+    OrganizationState
+  >((state) => state.organization);
+
+  useEffect(() => {
+    dispatch(fetchOrganization());
+  }, []);
+
+  if (isFetchingOrganizations) {
+    return <span>loading...</span>;
+  }
+
+  if (user?.role.id === 'USER' && orgId) {
+    const org = organizations.find((o) => o.id === orgId);
+    userOrganizations = [org];
+  }
+  userOrganizations = organizations;
```

Comment:
What are we trying to do here? We checked if user's role is `USER`, and if it is, we assigned a single organization to `userOrganizations`. And then we assigned all organizations we could fetch to the user anyway. So the user.role check above isn't really doing anything here.